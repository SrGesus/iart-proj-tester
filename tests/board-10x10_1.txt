VC	FD	FC	FC	LH	VC	FE	FB	BC	FD
BC	BC	BB	FB	FE	BE	BB	VB	BC	FC
LV	LV	BC	VB	VC	LH	VE	LH	LV	LH
FB	FC	BB	FB	VE	VE	VE	VD	FD	LH
VC	BE	BB	VD	VB	BB	BC	VE	FD	LH
FE	VB	FB	VE	BC	VE	BD	LV	FD	BB
FE	FC	LH	VC	FB	LV	BE	BB	LV	BC
BC	FD	FB	BE	FE	VB	BE	LV	FB	LH
VD	LV	VD	VD	LH	BC	BE	LV	VD	LH
FD	LH	BD	LV	LH	VD	FD	LH	VE	FB
