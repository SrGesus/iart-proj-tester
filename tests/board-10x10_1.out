VB	FE	FB	FD	LH	VE	FB	FD	BB	FE
BD	BB	BE	FB	FD	BC	BE	VB	BE	FB
LV	LV	BD	VC	VB	LH	VC	LV	LV	LV
FC	FC	BD	FE	VD	VE	VB	VC	FC	LV
VB	BB	BC	VE	VB	BC	BE	VB	FE	LV
FC	VD	FE	VD	BE	VB	BE	LV	FD	BE
FB	FD	LH	VE	FC	LV	BD	BC	LH	BE
BD	FE	FD	BE	FD	VC	BD	LH	FE	LV
VD	LH	VE	VD	LH	BB	BC	LH	VE	LV
FD	LH	BC	LH	LH	VC	FD	LH	VC	FC
